cli/tests/integrations/test_marathon_groups.py

```diff
@@ -123,6 +123,22 @@ def test_update_missing_field():
             "Possible properties are: ")
 
 
+def test_scale_group():
```

Comment:
This is great but let's also add negative tests. For example what happens when `<group-id>` doesn't exists, `<scale-factor>` is negative, `<scale-factor>` is not a flow (e.g. `dcos marathon group scale scale-group not_a_float`)